src/oh_component.cpp

```diff
@@ -169,7 +177,13 @@ void OHComponent::run(const double runToDate) {
         CNMVOC *
         ((1.0 * +current_nmvoc) -
          NMVOC_emissions.get(NMVOC_emissions.firstdate()).value(U_TG_NMVOC));
-    toh = a + b + c + d;
+    const double e =
+         CH2 *
+          ((1.0 * +current_h2) -
+           H2_emissions.get(H2_emissions.firstdate()).value(U_TG_H2));
+
+
+    toh = a + b + c + d + e;
```

Comment:
@bpbond do we have strong feelings on the use of e as a variable name?

Reply:
switch to f, we would like to be consistent with the notation above

Reply:
Single letter variables seem overly cryptic to me. Perhaps something like this: 
toh =  toh_ch4 + toh_nox ... + toch_h2
(or capitalize if that is your style).

Also, suggest putting a comment as such above:

// toh is a sum of coefficients in an exponential representation of OH lifetime
double toh = 0.0;

[Just so that readers are clear without reading the next page of code what this is.]

Reply:
+1 to the point about single-letter variables.